ee/tabby-ui/lib/stores/utils.ts

```diff
@@ -7,3 +10,52 @@ export const getChatById = (
   if (!Array.isArray(chats) || !chatId) return undefined
   return chats.find(c => c.id === chatId)
 }
+
+// @reference:
+// https://github.com/pmndrs/zustand/blob/main/docs/integrations/persisting-store-data.md#hydration-and-asynchronous-storages
+export const createStoreWithHydrated = ({
```

Comment:
Use generics to make the return value typed

Reply:
Updated to `export const createStoreWithHydrated = <T extends Record<string, any>>({`